region/lowernorfair/east.json

```diff
@@ -1667,7 +1730,7 @@
                 ]},
                 {"or": [
                   {"and": [
-                    { "cost": [{ "heatFrames": 680 }]},
+                    { "cost": [{ "heatFrames": 580 }]},
```

Comment:
the count was off?

Reply:
I moved 100 heatFrames into killing the Dessgeega at the new Obstacle C. Not the easiest balancing but hope it will do.